app/ledger.js

```diff
@@ -981,8 +986,12 @@ var stickyP = (publisher) => {
   var result = (siteSetting) && (siteSetting.get('ledgerPayments'))
 
   // NB: legacy clean-up
-  if ((typeof result === 'undefined') && (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined')) {
-    result = synopsis.publishers[publisher].options.stickyP
+  if (typeof result === 'undefined') {
+    if (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined') {
+      result = synopsis.publishers[publisher].options.stickyP
+    } else {
+      result = true
+    }
     appActions.changeSiteSetting(pattern, 'ledgerPayments', result)
   }
   delete synopsis.publishers[publisher].options.stickyP
```

Comment:
thanks, no crashes this time. However I followed STR and publisher is still auto-including for me as soon as it populates payments panel. 

I did a look inside publisherToggle (only part besides payments that changes publisher state) but state is only changed there if you hit urlbar icon, so it's must come from ledger itself.

IMO you might want something similar to this:
https://github.com/brave/browser-laptop/pull/7289/files#diff-9b959b675446ba95e1abe0b81e0f7a9bL566